### PR TITLE
Update CHANGELOG for 0.9.0 release

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Review by yperbasis — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,9 +8,59 @@ and this project adheres to [Semantic Versioning].
 
 ## [0.9.0] — unreleased
 
+In this release we have been focused on improving performance of the Baseline interpreter.
+The end result is that the Baseline is **26% faster** than in previous version 0.8.0
+and **18% faster** than the Advanced interpreter while having 
+over **8x smaller** code analysis cost. The Baseline is now the _default_ interpreter because
```
Comment (axic) [inline CHANGELOG.md:14]: I think the changes of the settings should be explained here as that is a "breaking" change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,9 +8,59 @@ and this project adheres to [Semantic Versioning].
...
+The end result is that the Baseline is **26% faster** than in previous version 0.8.0
+and **18% faster** than the Advanced interpreter while having 
+over **8x smaller** code analysis cost. The Baseline is now the _default_ interpreter because 
+it is simpler and has become better than the Advanced. 
+
+The Advanced also has got **4% faster** than in the previous version.
+
+All numbers are from running the "main" benchmark suite
+on 4.0 GHz Intel Haswell CPU, using the Clang 15 compiler.
+
```
Comment (axic) [inline CHANGELOG.md:21]: Would also highlight the gas refunds part here.

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,9 +8,59 @@ and this project adheres to [Semantic Versioning].
 
 ## [0.9.0] — unreleased
 
+In this release we have been focused on improving performance of the Baseline interpreter.
+The end result is that the Baseline is **26% faster** than in previous version 0.8.0
+and **18% faster** than the Advanced interpreter while having
```
[the remark this answers is not in the document]
Comment (chfast) [reply CHANGELOG.md]: In this release. Added "current".

File: CHANGELOG.md (hunk 1 above)
Comment (chfast) [reply CHANGELOG.md:14]: This is the first item in "Changed" section.

File: CHANGELOG.md (hunk 2 above)
Comment (chfast) [reply CHANGELOG.md:21]: Added.

**Review by axic — APPROVED** [review]

**Pull request merged**